app/models/item.rb

```diff
@@ -1,3 +1,9 @@
 class Item < ApplicationRecord
   belongs_to :supermarket
+  has_many :customer_items
+  has_many :customers, through: :customer_items
+
+  def customer_count
+    customers.count
```

Comment:
Nice work following strict MVC and encapsulating this logic (albeit simple) at the model level!